ratbag_emu/__main__.py

```diff
@@ -26,7 +27,7 @@ def main():
 
     # Run server
     server = connexion.FlaskApp(__name__,
-                                specification_dir='ratbag_emu/openapi/',
+                                specification_dir=pkg_resources.resource_filename('ratbag_emu', 'openapi/'),
```

Comment:
Is this change really related to the move from `ratbag-emu.py` to `__main__.py`? (hint: I have no ideas)

Reply:
Yes, we install the specification file as package data and `pkg_resources` allows us to fetch that file without going through hops finding the correct path.

Reply:
OK then